site/content/blog/dagger-and-kapp.md

```diff
@@ -0,0 +1,184 @@
+---
+title: kapp and dagger"
+slug: dagger-and-cloak
+date: 2022-07-30
+author: Renu Yarday
+excerpt: "kapp package for dagger"
+image: /img/kapp.svg
+tags: ['kapp', 'dagger']
+---
+
+# Running kapp in a Dagger pipeline
+
+In this article we will explore how to leverage kapp in a Dagger pipeline.
+
+### What is Dagger?
+Dagger is a portable devkit to build powerful CI/CD pipelines quickly and run them anywhere. Read more about Dagger [here](https://dagger.io/)
+
+### Introducing kapp package for dagger
+Do you want to observe your kubernetes deployments are going smoothly in your dagger pipeline? Then we highly recommend trying out [kapp deploy](https://github.com/dagger/dagger/tree/main/pkg/universe.dagger.io/alpha/kubernetes/kapp). kapp is now available as an alpha package with dagger and can be easily consumed in your CI/CD. 
+
+Below are the steps that one could use to add kapp to a dagger pipeline.
+
+### The micro services demo project
+Leveraging the well known [microservices demo](https://github.com/GoogleCloudPlatform/microservices-demo) to create dagger plan to deploy. The cloned and updated project to run in a local kind cluster is present [here](https://github.com/renuy/microservices-demo).
+
+This project's deployment manifest(configuration) for kubernetes is available as ./release/kubernetes-manifests.yaml
+
+### Build and run locally using dagger plan
+We need to have a dagger plan in place to deploy the application in the cluster. In this blog since the project is already created and build available as image, we will extend the project using the dagger plan to deploy it to a cluster (kind cluster).
```

Comment:
```suggestion
We need to have a Dagger plan in place to deploy the application in the cluster. In this blog, since the project is already created and build available as image, we will extend the project using the Dagger plan to deploy it to a cluster (kind cluster).
```